src/libs/ReportUtils.js

```diff
@@ -3769,6 +3769,14 @@ function getIOUReportActionDisplayMessage(reportAction) {
     return displayMessage;
 }
 
+/**
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function isReportDraft(report) {
+    return lodashGet(report, 'stateNum') === CONST.REPORT.STATE_NUM.OPEN && lodashGet(report, 'statusNum') === CONST.REPORT.STATUS.OPEN;
+}
```

Comment:
Could we actually also add a check for the expense type, ie this has to be expense report in order to be a draft, its extra check but I think it should be better